compiler/crates/relay-typegen/tests/generate_typescript/fixtures/updatable-operation-plural-field-with-spreads.graphql

```diff
@@ -0,0 +1,9 @@
+query UpdatableQuery @updatable {
+  nodes(ids: [4]) {
+    ...Updatable_user
+  }
+}
+
+fragment Updatable_user on User @assignable {
+  __typename
+}
```

Comment:
Internal lint rules will reject the diff if trailing line endings are not included :(

Could you add one here?